tests/component_handlers/helm_wrapper/test_utils.py

```diff
@@ -1,29 +1,35 @@
-from kpops.component_handlers.helm_wrapper.utils import trim_release_name
+import hashlib
 
+from kpops.component_handlers.helm_wrapper.utils import (
+    create_helm_release_name,
+)
 
-def test_trim_release_name_with_suffix():
-    name = trim_release_name(
-        "example-component-name-too-long-fake-fakefakefakefakefake-clean",
-        suffix="-clean",
+
+def test_helm_release_name_for_long_names():
+    long_release_name = "example-component-name-too-long-fake-fakefakefakefakefake"
+    actual_release_name = (
+        "example-component-name-too-long-fake-fakefakef-"
+        + hashlib.sha1(long_release_name.encode("utf-8")).hexdigest()[:4]
```

Comment:
instead of repeating the hashing logic here, I would prefer a constant actual value.